src/test/java/SubsystemTest.java

```diff
@@ -0,0 +1,33 @@
+import static org.junit.Assert.*;
+
+import edu.wpi.first.hal.HAL;
+import frc.robot.subsystems.TurretSubsystem;
+import com.ctre.phoenix.motorcontrol.VictorSPXSimCollection;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SubsystemTest {
+
+    TurretSubsystem m_turretSubsystem;
+
+    @Before
+    public void setup(){
+    assert HAL.initialize(500, 0);
```

Comment:
Buradaki 500 ne anlam ifade ediyor bir comment yazabilir miyiz?

Reply:
500 initialize olması için verdiğimiz timeout, ekliyorum commenti

Reply:
Şahane!